packages/orbit-components/src/Checkbox/Checkbox.ct.tsx

```diff
@@ -1,12 +1,22 @@
 import * as React from "react";
 import { test, expect } from "@playwright/experimental-ct-react";
 
-import CheckboxStory from "./Checkbox.ct-story";
+import { CheckboxStory, CheckboxWithTooltipStory } from "./Checkbox.ct-story";
 
 test.describe("visual Checkbox", () => {
   test("Checkbox", async ({ mount }) => {
```

Comment:
I was also considering this, but I wanted to avoid a huge number of screenshots. But it makes more sense from the point of the code "readability". 🤔 👍

Reply:
and what do you think about the `test.skip` command? I haven't found any better way how to solve this problem.

Reply:
I like the `test.skip`, I was at first confused as I didn't know you can skip the test based on the condition 👍 TIL

But I am not sure about the condition in this case. I see we are setting different devices/viewports in the config, isn't there a way to access this info inside the test and act on that info?